templates/shared/secrets/add_list.tmpl

```diff
@@ -32,7 +32,7 @@
 		{{range .Secrets}}
 		<div class="item">
 			<div class="right floated content">
-				<button class="ui red tiny button delete-button" data-url="{{$.Link}}/delete" data-id="{{.ID}}">
+				<button class="ui red tiny button delete-button" data-url="{{if $.isFromOrg}}{{$.Link}}/delete{{else}}{{$.Link}}/secrets/delete{{end}}" data-id="{{.ID}}">
```

Comment:
I would suggest to use the same URL pattern to avoid the unnecessary `{{if}}`

Reply:
The urls are actually the same. This `if` exists because for organization, the nav bar is vertical, and when runners/secrets [submenu](https://fomantic-ui.com/collections/menu.html#sub-menu) is clicked,  they are two pages 

<img width="720" alt="Screen Shot 2023-04-19 at 15 34 41" src="https://user-images.githubusercontent.com/17645053/233038271-72c8314e-857e-40a5-a2e6-da185eade695.png">

While for repo/user/admin, the nav bar is horizontal, and runners and secrets share the same page

<img width="1415" alt="Screen Shot 2023-04-19 at 15 34 09" src="https://user-images.githubusercontent.com/17645053/233038628-e8dbc289-2df6-4d3d-b256-d6a5e8f75aee.png">

So resulting in different `{{$.Link}}` . For org level, `{{$.Link}}` will be like "<host>/org/testOrg/settings/actions/runners", while for user/admin/repo level, `{{$.Link}}` will be like"<host>/org/testOrg/settings/actions"

Reply:
It's unmaintainable if you use a lot of `if` in sub templates.

Either take a look at how the `oauth` shared templates work, they require the callers to have the same URL patterns.

Or, take a look at how `combomarkdowneditor` work, it defines "attributes", callers prepare the URLs and pass-in.